### PR TITLE
Initial set up for boosktore 

### PR DESCRIPTION
**Pull request by mandelatuks**
**In this milestone i did the following:**

 -Initialise React app.
-  Create a directory for your components.
- Add React Router for navigation
- Books - the default view:
Should display the list of books (empty at this point but it should be ready for the data) with the Remove button (no funcionality yet).
- Create a component called Book for displaying a single book (receive a title and an author as prop) and reuse it in a component that displays a list of books.
- Form for adding a book (no functionality yet).
- Create a separate component for this form 
- Categories page

- Styling is not required at this point.

### CODE REVIEW

File: src/components/CategoryPage.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const CategoryPage = () => (
+  <div>
+    <h1>Coming Soon!!</h1>
```
Comment (ignatius22) [inline src/components/CategoryPage.js]: - [x] Let's follow the guidelines by displaying only a button with text "Check status" here.

Comment (mandelatuks) [reply src/components/CategoryPage.js]: @ignatius22 
Done

Comment (mandelatuks) [reply src/components/CategoryPage.js]: @ignatius22 
Done

**Review by ignatius22 — CHANGES_REQUESTED** [review]
Hi @mandelatuks, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [ ] should display only a button with text "Check status"

## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Wes-Isaac — APPROVED** [review]
Hi @mandelatuks, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Linters are all green ✔️.
- Project works as expected 🚀.
- README looks professional 😮.

## Optional suggestions

- [ ] [OPTIONAL] Please always include professional readme in your feature branches.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**